docs/development-environment.md

```diff
@@ -82,7 +82,11 @@ Before you get started, we recommend that you set up a public SSH key setup with
 
 Fork this repository to your own GitHub account and clone it to your local machine, as explained [in this guide](https://guides.github.com/activities/forking/). **If you are an Automattician, you can clone the repository directly.**
 
-If you use [our Docker setup](../tools/docker/README.md), you can now move on to the next step. If you use a different setup, you'll first need to create symlinks from the plugin directory in your local installation of WordPress to each of the plugins' directories in the monorepo (under `projects/plugins/`).
+Note that the Monorepo should not be cloned into the WordPress plugins directory (you will see a warning on your plugins page in that case saying that the Jetpack Monorepo is not a plugin and shouldn't be installed as one). 
```

Comment:
The ordering seems off here. If they're using our Docker setup, they don't really need to know the stuff in this new paragraph. Would it serve the purpose to move this after the next two paragraphs instead?

Reply:
That would work, I think! I know I had another Docker installation with WordPress in it when I set things up back in January, and without knowing much about Docker could have attempted to clone within the plugin directory.

I think as long as the info is in this section that would help cover any edge cases like potential past me could have found myself in, so I'll move it down 👍